README.md

```diff
@@ -1,214 +1,214 @@
-# Label Studio &middot; ![GitHub](https://img.shields.io/github/license/heartexlabs/label-studio?logo=heartex) ![label-studio:build](https://github.com/heartexlabs/label-studio/workflows/label-studio:build/badge.svg) ![code-coverage](https://github.com/heartexlabs/label-studio/blob/master/.github/test-coverage.svg) ![GitHub release](https://img.shields.io/github/v/release/heartexlabs/label-studio?include_prereleases) &middot;
+# Label Studio 
+
+&middot; ![GitHub](https://img.shields.io/github/license/heartexlabs/label-studio?logo=heartex) ![label-studio:build](https://github.com/heartexlabs/label-studio/workflows/label-studio:build/badge.svg) ![code-coverage](https://github.com/heartexlabs/label-studio/blob/master/.github/test-coverage.svg) ![GitHub release](https://img.shields.io/github/v/release/heartexlabs/label-studio?include_prereleases) &middot;
 
 [Website](https://labelstud.io/) • [Docs](https://labelstud.io/guide/) • [Twitter](https://twitter.com/heartexlabs) • [Join Slack Community <img src="https://app.heartex.ai/docs/images/slack-mini.png" width="18px"/>](https://join.slack.com/t/label-studio/shared_invite/zt-cr8b7ygm-6L45z7biEBw4HXa5A2b5pw)
 
-<br/>
 
-**Label Studio is a swiss army knife of data labeling and annotation tools :v:**
- 
-Try it now in a **[running app](https://app.labelstud.io)** and check out the **[introductory post](https://towardsdatascience.com/introducing-label-studio-a-swiss-army-knife-of-data-labeling-140c1be92881#3907-fd502dc24c8d)**.
- 
-Its purpose is to help you label different types of data using a simple interface with a standardized output format. You're dealing with the custom dataset and thinking about creating your tool? Don't - using Label Studio, you can save time and create a custom tool and interface in minutes. 
+## What is Label Studio?
 
-<br/>
+Label Studio lets you label data types like audio, text, images, videos, and time series with a straightforward interface and standardized output formats. 
 
-![Label Studio](https://raw.githubusercontent.com/heartexlabs/label-studio/master/images/annotation_examples.gif)
+- [Try out Label Studio](#try-out-label-studio)
+- [What you get from Label Studio](#what-you-get-from-label-studio)
+- [Included templates for labeling data in Label Studio](#included-templates-for-labeling-data-in-label-studio)
+- [Integrate with machine learning models](#integrate-with-machine-learning-models)
+- [Integrate Label Studio with your machine learning tools](#integrate-label-studio-with-your-machine-learning-tools)
```

Comment:
what's the key difference between these 2 last ones?

Reply:
the first one covers setting up an ML backend basically, the second one is about the different parts of label studio that you can integrate with, so maybe would be better tied in with the ecosystem section too. Maybe this would be clearer?
```suggestion
- [Set up machine learning models with Label Studio](#set-up-machine-learning-models-with-Label-Studio)
- [Integrate Label Studio with your existing tools](#integrate-label-studio-with-your-existing-tools)
```
and then fold the ecosystem content into the tools section too?

Reply:
I did this, but left ecosystem separate because I think the audience is perhaps slightly different